--- qt-everywhere-opensource-src-5.2.1/qtbase/src/angle/src/libGLESv2/libGLESv2.pro	2013-12-09 02:09:53.000000000 +0900
+++ qt-everywhere-opensource-src-5.2.1/qtbase/src/angle/src/libGLESv2/libGLESv2.pro	2014-01-04 14:18:52.400101600 +0900
@@ -181,7 +181,7 @@
 SHADER_DIR = $$OUT_PWD/renderer/shaders/compiled
 
 for (ps, PIXEL_SHADERS_BLIT) {
-    fxc_ps_$${ps}.commands = $$FXC /nologo /E $$ps /T ps_2_0 /Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
+    fxc_ps_$${ps}.commands = $$FXC //nologo //E $$ps //T ps_2_0 //Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
     fxc_ps_$${ps}.output = $$SHADER_DIR/$${ps}.h
     fxc_ps_$${ps}.input = PS_BLIT_INPUT
     fxc_ps_$${ps}.dependency_type = TYPE_C
@@ -190,7 +190,7 @@
     QMAKE_EXTRA_COMPILERS += fxc_ps_$${ps}
 }
 for (ps, PIXEL_SHADERS_PASSTHROUGH) {
-    fxc_ps_$${ps}.commands = $$FXC /nologo /E PS_$$ps /T ps_4_0 /Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
+    fxc_ps_$${ps}.commands = $$FXC //nologo //E PS_$$ps //T ps_4_0 //Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
     fxc_ps_$${ps}.output = $$SHADER_DIR/$${ps}11ps.h
     fxc_ps_$${ps}.input = PASSTHROUGH_INPUT
     fxc_ps_$${ps}.dependency_type = TYPE_C
@@ -199,7 +199,7 @@
     QMAKE_EXTRA_COMPILERS += fxc_ps_$${ps}
 }
 for (ps, PIXEL_SHADERS_CLEAR) {
-    fxc_ps_$${ps}.commands = $$FXC /nologo /E PS_$$ps /T ps_4_0 /Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
+    fxc_ps_$${ps}.commands = $$FXC //nologo //E PS_$$ps //T ps_4_0 //Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
     fxc_ps_$${ps}.output = $$SHADER_DIR/$${ps}11ps.h
     fxc_ps_$${ps}.input = CLEAR_INPUT
     fxc_ps_$${ps}.dependency_type = TYPE_C
@@ -208,7 +208,7 @@
     QMAKE_EXTRA_COMPILERS += fxc_ps_$${ps}
 }
 for (vs, VERTEX_SHADERS_BLIT) {
-    fxc_vs_$${vs}.commands = $$FXC /nologo /E $$vs /T vs_2_0 /Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
+    fxc_vs_$${vs}.commands = $$FXC //nologo //E $$vs //T vs_2_0 //Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
     fxc_vs_$${vs}.output = $$SHADER_DIR/$${vs}.h
     fxc_vs_$${vs}.input = VS_BLIT_INPUT
     fxc_vs_$${vs}.dependency_type = TYPE_C
@@ -217,7 +217,7 @@
     QMAKE_EXTRA_COMPILERS += fxc_vs_$${vs}
 }
 for (vs, VERTEX_SHADERS_PASSTHROUGH) {
-    fxc_vs_$${vs}.commands = $$FXC /nologo /E VS_$$vs /T vs_4_0 /Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
+    fxc_vs_$${vs}.commands = $$FXC //nologo //E VS_$$vs //T vs_4_0 //Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
     fxc_vs_$${vs}.output = $$SHADER_DIR/$${vs}11vs.h
     fxc_vs_$${vs}.input = PASSTHROUGH_INPUT
     fxc_vs_$${vs}.dependency_type = TYPE_C
@@ -226,7 +226,7 @@
     QMAKE_EXTRA_COMPILERS += fxc_vs_$${vs}
 }
 for (vs, VERTEX_SHADERS_CLEAR) {
-    fxc_vs_$${vs}.commands = $$FXC /nologo /E VS_$$vs /T vs_4_0 /Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
+    fxc_vs_$${vs}.commands = $$FXC //nologo //E VS_$$vs //T vs_4_0 //Fh ${QMAKE_FILE_OUT} ${QMAKE_FILE_NAME}
     fxc_vs_$${vs}.output = $$SHADER_DIR/$${vs}11vs.h
     fxc_vs_$${vs}.input = CLEAR_INPUT
     fxc_vs_$${vs}.dependency_type = TYPE_C
