---
 mingw-w64-headers/include/wrl/client.h | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/mingw-w64-headers/include/wrl/client.h b/mingw-w64-headers/include/wrl/client.h
index 83b4cb3..618bbf2 100644
--- a/mingw-w64-headers/include/wrl/client.h
+++ b/mingw-w64-headers/include/wrl/client.h
@@ -88,7 +88,7 @@ namespace Microsoft {
             }
 
             template<class U>
-            ComPtr(const ComPtr<U> &other) throw() : ptr_(other.ptr_) {
+            ComPtr(const ComPtr<U> &other) throw() : ptr_(other.Get()) {
                 InternalAddRef();
             }
 
@@ -98,8 +98,7 @@ namespace Microsoft {
             }
 
             template<class U>
-            ComPtr(ComPtr<U>&& other) throw() : ptr_(other.ptr_) {
-                other.ptr_ = nullptr;
+            ComPtr(ComPtr<U>&& other) throw() : ptr_(other.Detach()) {
             }
 
             ~ComPtr() throw() {
-- 
2.9.3
