diff --git a/qtbase/src/gui/opengl/qopengltexturehelper.cpp b/qtbase/src/gui/opengl/qopengltexturehelper.cpp
index 29cecf0..5ba51cc 100644
--- a/qtbase/src/gui/opengl/qopengltexturehelper.cpp
+++ b/qtbase/src/gui/opengl/qopengltexturehelper.cpp
@@ -170,7 +170,7 @@ QOpenGLTextureHelper::QOpenGLTextureHelper(QOpenGLContext *context)
     GetTexParameteriv = ::glGetTexParameteriv;
     GetTexParameterfv = ::glGetTexParameterfv;
     GetTexImage = 0;
-    TexImage2D = ::glTexImage2D;
+    TexImage2D = reinterpret_cast<void (*)(GLenum, GLint, GLint, GLsizei, GLsizei, GLint, GLenum, GLenum, const GLvoid *)>(::glTexImage2D);
     TexImage1D = 0;
     TexParameteriv = ::glTexParameteriv;
     TexParameteri = ::glTexParameteri;
diff --git a/qtbase/src/plugins/platforms/eglfs/eglfs.pri b/qtbase/src/plugins/platforms/eglfs/eglfs.pri
index 6f463ba..243503c 100644
--- a/qtbase/src/plugins/platforms/eglfs/eglfs.pri
+++ b/qtbase/src/plugins/platforms/eglfs/eglfs.pri
@@ -5,8 +5,8 @@ DEFINES += MESA_EGL_NO_X11_HEADERS
 
 # Uncomment these to enable the X hooks, allowing to test the platform
 # plugin in a regular X11 environment (as long as EGL is available).
-# EGLFS_PLATFORM_HOOKS_SOURCES += qeglfshooks_x11.cpp
-# LIBS += -lX11 -lX11-xcb -lxcb
+EGLFS_PLATFORM_HOOKS_SOURCES += qeglfshooks_x11.cpp
+LIBS += -lX11 -lX11-xcb -lxcb
 
 # Uncomment these to enable the KMS hooks.
 # EGLFS_PLATFORM_HOOKS_SOURCES += qeglfshooks_kms.cpp
