--- eglplatform_orig.h	2010-07-30 23:13:05.000000000 +0900
+++ eglplatform.h	2014-11-18 15:49:52.297306751 +0900
@@ -75,6 +75,14 @@
 #include <X11/Xlib.h>
 #include <X11/Xutil.h>
 
+#ifdef QT5_EGL
+/* X11のヘッダーからQtのシンボルと重複するマクロを解除 */
+#undef CursorShape
+#undef None
+#undef Expose
+#undef Status
+#endif
+
 typedef Display *EGLNativeDisplayType;
 typedef Pixmap   EGLNativePixmapType;
 typedef Window   EGLNativeWindowType;
