--- CMakeLists.txt	2016-09-09 06:49:34.000000000 +0900
+++ CMakeLists.txt.bak	2017-04-23 18:57:45.607034000 +0900
@@ -1,6 +1,7 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.11)
 
 # Project properties
+LINK_DIRECTORIES(/mingw64/local/lib)
 PROJECT(qtspell)
 SET(QTSPELL_VERSION 0.8.2)
 
@@ -147,3 +148,5 @@
     COMMAND git archive --format=tar --prefix=${CMAKE_PROJECT_NAME}-${QTSPELL_VERSION}/ HEAD | xz > ${CMAKE_BINARY_DIR}/${CMAKE_PROJECT_NAME}-${QTSPELL_VERSION}.tar.xz
     WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
 )
+
+TARGET_LINK_LIBRARIES(qtspell Qt5Gui png qtfreetype qtharfbuzz libEGL libGLESv2 translator preprocessor d3d9 dxguid UxTheme Qt5Core qtpcre ws2_32 winmm)
